.github/workflows/pr.yaml

```diff
@@ -6,6 +6,7 @@ env:
 jobs:
   run-image:
     runs-on: ubuntu-latest
+    timeout-minutes: 15
```

Comment:
just to be sure the the commit message and the title if the PR both say 30 minutes :scream: . Can you update the messages so they match the actual timeout?

Reply:
yes lol I changed my mind

Reply:
commit message still says 30 minutes :scream:

Reply:
@nottrobin can you update the commit message?

Reply:
HAPPY NOW?!!